open_fdd/quality.py

```
@@ -0,0 +1,330 @@
+            sent = _is_sentinel(num, sentinels)
+            reason = reason.mask(sent, REASON_SENTINEL)
+            valid = valid & ~sent
+            if bounds is None:
+                bounds = (0.0, 100.0)
+            lo, hi = bounds
+            out_of = pd.Series(False, index=raw.index)
+            if lo is not None:
+                out_of = out_of | (num < lo)
+            if hi is not None:
+                out_of = out_of | (num > hi)
+            # 0–1 fraction commands are in range
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)